CONTRIBUTING.md

```diff
@@ -78,6 +78,8 @@ To point the app to staging, create a .env file and set the `ENV` variable to `s
 cp .env.example .env
 ```
 
+To use the geolocation service when running on the web version, you need to have an API key from Opencage. You can obtain an API key for free by registering on their website, and there is no need to register a payment method for this. After obtaining the API key, add it to the `OPENCAGE_KEY` environment variable.
```

Comment:
could you move this above? close to the backend section

Reply:
It was unclear whether this content should be in the "Running the infrastructure & backend with Docker" section or the "Running without Docker" section.